client_cli/README.md

```diff
@@ -43,71 +49,56 @@ Full description and list of commands detailed in `./iroha_client_cli --help`.
 
 ### Create new Domain
 
-Let's start with domain creation. We need to provide `register` command first, 
-following by entity type (domain in our case) and list of required parameters.
-For domain entity we only need `id` parameter as a string.
+Let's start with domain creation. To create a domain, you need to specify the entity type first (`domain` in our case) and then the command (`register`) with a list of required parameters.
+For the `domain` entity, you only need to provide the `id` argument as a string that doesn't contain the `@` and `#` symbols.
 
 ```bash
 ./iroha_client_cli domain register --id="Soramitsu"
 ```
 
+Now you have a domain without any accounts.
+
 ### Create new Account
 
-Right now we have the only domain without any accounts, let's fix it.
-Like in the previous example, we need to define account name, it is done using `id` flag.
-We also give a `key` argument with account's public key as a double-quoted
-string value.
+Let's create a new account. Like in the previous example, specify the entity type (`account`) and the command (`register`). Then define the account name as the value of the `id` argument.
+Additionally, you need to provide the `key` argument with the account's public key as a double-quoted multihash representation of the key. Providing an empty string also works (but is highly discouraged), while omitting the argument altogether will produce an error.
 
 ```bash
 ./iroha_client_cli account register --id="White Rabbit@Soramitsu" --key=""
 ```
 
 ### Mint Asset to Account
 
-Okay, it's time to give something to our Account. We will add some Assets quantity to it.
-This time we need to register an Asset Definition first and then add some Assets to the account.
-As you can see, we use new command `asset` and it's subcommands `register` and `mint`. 
-Every asset has its own value type, here we define domain as quantity (integer/number).
+It's time to give something to the Account you created. Let's add some Assets of the type `Quantity` to the account.
+
+To do so, you must first register an Asset Definition and only then add some Assets to the account. Specify the `asset` entity and then use the `register` and `mint` commands respectively. 
+	
+Every asset has its own value type. In this example, it is defined as `Quantity`, a 32-bit unsigned integer. We also support `BigQuantity` and `Fixed`, which are a 128-bit unsigned integer and a 64-bit fixed-precision binary fraction.
```

Comment:
There is also another type called [`Store`](https://github.com/hyperledger/iroha/blob/49407ff210a96c63de41038d9ee68007e5f02098/data_model/src/asset.rs#L189). If I understand it right, it can be used to create *NFT* or some other things I can't find an example of